modules/odc-uninstalling-web-terminal.adoc

```diff
@@ -7,34 +7,64 @@
 
 Uninstalling the Web Terminal Operator does not remove any of its custom resource definitions (CRDs) or managed resources that are created when the Operator is installed. These components must be manually uninstalled for security purposes. Removing these components also allows you to save cluster resources by ensuring that terminals do not idle when the Operator is uninstalled.
 
 .Prerequisites
 * Access to an {product-title} cluster using an account with `cluster-admin` permissions.
 
-== Deleting the web terminal components and custom resources
+== Removing the Web Terminal Operator and the custom resources that support it
 
-Use the CLI to delete the CRs that are created during installation of the  Web Terminal Operator.
+Use the CLI to delete any existing web terminals and CRs that were created during the installation of the Web Terminal Operator.
+
+[NOTE]
+====
+Prior to {product-title} 4.8, the Web Terminal Operator used different CRDs to provide Web Terminal capabilities. To uninstall versions 1.2.1 and below of the Web Terminal Operator, refer to the documentation for {product-title} 4.7.
+====
 
 .Procedure
-. Run the following commands to ensure that all `DevWorkspace` CRs are removed along with their related Kubernetes objects, such as deployments.
+. Uninstall the Web Terminal Operator via the web console:
+.. In the *Administrator* perspective of the web console, navigate to *Operators -> Installed Operators*.
+.. Scroll the filter list or type a keyword into the *Filter by name* box to find the *Web Terminal* Operator.
+.. Click the Options menu {kebab} for the Web Terminal Operator, and then select *Uninstall Operator*.
+.. In the *Uninstall Operator* confirmation dialog box, click *Uninstall* to remove the Operator, Operator deployments, and pods from the cluster. The Operator stops running and no longer receives updates.
++
+. Run the following commands to ensure that all `DevWorkspace` CRs created for Web Terminals are removed.
 +
 [source,terminal]
 ----
-$ oc delete devworkspaces.workspace.devfile.io --all-namespaces --all --wait
+$ oc delete devworkspaces.workspace.devfile.io --all-namespaces \
+    --selector 'console.openshift.io/terminal=true' --wait
 ----
 +
 [source,terminal]
 ----
-$ oc delete workspaceroutings.controller.devfile.io --all-namespaces --all --wait
+$ oc delete devworkspacetemplates.workspace.devfile.io --all-namespaces \
+    --selector 'console.openshift.io/terminal=true' --wait
 ----
+
+== Deleting the DevWorkspace Operator dependency
+
+Use the CLI to delete the custom resource definitions (CRDs) and additional resources that are created during installation of the Web Terminal Operator.
+
+[NOTE]
+====
+The DevWorkspace Operator functions as a standalone operator and may be required as a dependency of other operators installed on the cluster (for example, the CodeReady Workspaces Operator may depend on it). Only follow the steps below if you are sure the DevWorkspace Operator is no longer needed.
```

Comment:
/of other/for other
Operators
Only follow the steps below if.../Follow the steps below only if...